src/routes/home/components/AnnouncementDialog.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <BaseDialog
+    id="announcement-dialog"
+    v-model="show"
+    variant="light"
+    :opacity="1"
+    data-cy="announcement-dialog"
+    scrollable
+    :max-width="750"
+  >
+    <template #activator>
+      <v-btn
+        color="newSecondary"
+        width="full"
+        rounded="0"
+        block
+      >
+        Announcement Header
+      </v-btn>
+    </template>
+    <template #title>
+      <div color="primary" class="w-100">
+        <div class="d-flex justify-space-between w-100">
+          <h1 class="mb-4">
+            Generic Header
+          </h1>
+          <v-btn
+            variant="text"
+            icon
+            size="x-large"
+            @click="close"
+          >
+            <v-icon icon="mdi-close" />
+          </v-btn>
+        </div>
+      </div>
+    </template>
+    <template #body>
+      <div class="d-flex justify-center my-4">
+        <GameCard
+          v-for="card in displayCards"
+          :key="card.suit + card.rank"
+          :suit="card.suit"
+          :rank="card.rank"
+          class="card mb-10"
+          :high-elevation="true"
+        />
```

Comment:
I'm not sure whether we want this as a standard thing in the announcement. Maybe we do it as optional by v-if'ing the wrapper, and optionally add an image above that

```
      <img v-if="imgSrc" :src="/img/${imgSrc}">
      <div v-if="displayCards.length" class="d-flex justify-center my-4">
        <GameCard
          v-for="card in displayCards"
          :key="card.suit + card.rank"
          :suit="card.suit"
          :rank="card.rank"
          class="card mb-10"
          :high-elevation="true"
        />
```